examples/postgres/test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -x
+
+./build.sh
+
+# test it
+docker stop test-postgres-func
+docker rm test-postgres-func
+
+docker run -p 5432:5432 --name test-postgres-func -d postgres
+sleep 5s
+
+docker run --rm -i --link test-postgres-func:postgres postgres psql -h postgres -U postgres -c 'CREATE TABLE people (first TEXT, last TEXT, age INTEGER);'
+
+RECORD1='{
+    "first": "John",
+    "last": "Smith",
+    "age": 30
+}'
+echo $RECORD1 | docker run --rm -i -e SERVER=postgres:5432 -e COMMAND=INSERT -e TABLE=people --link test-postgres-func:postgres iron/func-postgres
```

Comment:
I modified my local 'fn run' to support passing through '--link' flags, although this feature is apparently deprecated, so this may not be the right approach.

I also believe I found a bug [here](https://github.com/iron-io/functions/blob/master/fn/run.go#L86), where kv[1] should be used in place of os.Getenv(). (Or perhaps a check for kv[1], with a fallback to lookup)

Are either of these changes desired?  If so, should I push the changes here, or open new PRs and keep things organized?